### ISSUE TITLE
Conflict with dependency

### ISSUE THREAD

**Issue by araqaee** [fabian-hiller/modular-forms#115]
It seems modular forms uses previous version of Qwik but the current version is 1.2.6. Because of this NPM doesn't find the previous version and it gives an error during NPM install.

**Comment by fabian-hiller** [comment]
This is not quite correct. As you can see from [this issue](https://github.com/BuilderIO/qwik/issues/4789), Qwik version 1.2.X and higher makes problems with Modular Forms. For this reason, I require Qwik as a dependency in `package.json` in version v1.1.5, as that is the last version I know of that does not cause problems. So you have to downgrade the Qwik version manually.

```bash
npm i -D @builder.io/qwik@1.1.5 @builder.io/qwik-city@1.1.5 eslint-plugin-qwik@1.1.5
```

**Comment by AmirSa12** [comment]
@fabian-hiller , I'm facing the same issue! The command lines are throwing an error. Could you please suggest an alternative solution? Thanks!
`09:04:37.501	npm ERR! code ERESOLVE
09:04:37.504	npm ERR! ERESOLVE unable to resolve dependency tree
09:04:37.504	npm ERR! 
09:04:37.504	npm ERR! While resolving: 
09:04:37.504	npm ERR! Found: @builder.io/qwik@1.2.6
09:04:37.504	npm ERR! node_modules/@builder.io/qwik
09:04:37.505	npm ERR!   dev @builder.io/qwik@"^1.2.6" from the root project
09:04:37.505	npm ERR! 
09:04:37.505	npm ERR! Could not resolve dependency:
09:04:37.505	npm ERR! peer @builder.io/qwik@"1.1.5" from @modular-forms/qwik@0.19.1
09:04:37.505	npm ERR! node_modules/@modular-forms/qwik
09:04:37.505	npm ERR!   @modular-forms/qwik@"^0.19.1" from the root project
09:04:37.505	npm ERR! 
09:04:37.505	npm ERR! Fix the upstream dependency conflict, or retry
09:04:37.505	npm ERR! this command with --force, or --legacy-peer-deps
09:04:37.506	npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
09:04:37.506	npm ERR! 
09:04:37.506	npm ERR! See /opt/buildhome/.npm/eresolve-report.txt for a full report.
09:04:37.507	
09:04:37.507	npm ERR! A complete log of this run can be found in:
09:04:37.507	npm ERR!     /opt/buildhome/.npm/_logs/2023-07-27T07_04_36_319Z-debug-0.log
09:04:37.513	Error during NPM install
09:04:37.518	Failed: build command exited with code: 1
09:04:38.430	Failed: error occurred while running build command`

**Comment by sajjad-soudani** [comment]
@fabian-hiller I have this issue too

**Comment by fabian-hiller** [comment]
Downgrade to Qwik 1.1.5. You have to make sure to remove `^` from `^1.1.5` in your `package.json`.

**Comment by AmirSa12** [comment]
Will you make it compatible with the latest version of Qwik?
Many users would prefer not to downgrade.

**Comment by fabian-hiller** [comment]
Yes, of course. The problem is probably Qwik and not Modular Forms. I have to wait until the bug is fixed in Qwik.

**Comment by fabian-hiller** [comment]
Should be fixed in v0.20.0. Make sure, this two imports are imported in the following order in `entry.preview.tsx`:

```ts
import qwikCityPlan from "@qwik-city-plan"; 
import render from "./entry.ssr";
```
